contents/blog/improving-posthog-deployments.md

```diff
@@ -0,0 +1,123 @@
+---
+date: 2022-03-22
+title: "Improving PostHog deployments"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/launch-week-teaser.jpeg # TODO
+featuredImageType: full
+categories: ["Engineering", "Product updates", "Launch week"]
+author: ["harry-waye", "guido-iaquinti"]
+---
+
+When PostHog was born in 2020, it was a simple Python application (Django + Celery) backed by a PostgreSQL datastore. Troubleshooting was easy, while the low barrier of entry meant fast adoption and more feedback on where to take the product.
+
+This was great at the start, but it couldn't last. More customers meant more data to ingest and analyze. Our ingestion couldn’t keep up with the volumes of events we were receiving, and our event queries using PostgreSQL were slow. If we wanted PostHog to scale, we had to revisit our stack:
+
+TODO: insert slowly moving hedgehog image
```

Comment:
Same here

Reply:
Don't think we have anything like this, so we'll have to leave out for now